### ISSUE TITLE
Continue previous runs in new folder

### ISSUE THREAD

**Issue by joaomcteixeira** [haddocking/haddock3#382]
We want to take an haddock3 run and continue it in a different direction in another run folder. There are several possibilities this can be done, and they can all be implemented together:

- [ ] take the whole run or part of it, copy it to a new folder, and start it from a new configuration that defines only the new steps.
- [ ] take only the topology folder and the desired step and go forward as in the previous point
- [ ] give a folder through an  `haddock3` command parameter and a config file so the run continues.

**Comment by joaomcteixeira** [comment]
Although we have implemented #378 and #383, I wonder if that flexibility in the module's folder number prefixes will perturb the functionalities we want to add here. The problem may arise if we have a run with 8 modules and than we extend it for a run with 12 modules. How are the folders and files from the previous run be handled? We will have a mixture of folders starting with `0` and without. This example would result in,

```
0_topoaa
1_...
1_...
1_...
1_...
1_...

**Comment by rvhonorato** [comment]
This already exists, doesn't it?

The below works just fine:

```bash
$ haddock3 workflow-with-two-steps.cfg
---
$ mkdir ~/other-folder
$ cp -r data ~/other-folder
$ cp workflow-with-two-steps.cfg ~/other-folder/workflow-with-five-steps.cfg
$ cd ~/other-folder
# edit workflow-with-five-steps.cfg and add the new steps
$ haddock3 --restart 2 workflow-with-five-steps.cfg
```

**Comment by joaomcteixeira** [comment]
Yes. That part exists. But there are some variations of that protocol that I am unsure will work straight with the current implementation. I want to use this issue to review all the possibilities, implement what's left, and make docs about it also. The example you show definitively works already.

**Comment by joaomcteixeira** [comment]
# Case 1

*describing the behavior for case 1*

1. take a completed run, for example `protein-protein` docking:

```
0_topoaa
1_rigidbody
2_caprieval
3_seletop
4_flexref
5_caprieval
6_emref
7_caprieval
8_clustfcc
data
log
```

Now we want to take `4_flexref` to another place and continue a different run from there. How should we handle it? A possibility is:

Copy `0_topoaa` and `4_flexref` to a new folder:

```
0_topoaa
4_flexref
```

Reprepare the configuration file only with the steps we need onwards:

```toml
[caprieval]
reference_fname = "data/e2a-hpr_1GGR.pdb"

[emref]
tolerance = 20
ambig_fname = "data/e2a-hpr_air.tbl"

[clustfcc]
```

And the magic should happen from here.
